app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -27,6 +27,20 @@ const EMOJI_LIST_OFFSET_LEFT = 8;
 const EMOJI_LIST_OFFSET_TOP = 8;
 const QUERY_MIN_LENGTH = 1;
 
+// DO NOT USE COLON WITH LOWERCASE LETTERS IN THE KEYS, or you will prevent searching emojis.
+// For example, while :D should be replaced with unicode symbol, :d should allow searching for :dancer:
+const EMOJI_INLINE_REPLACEMENT = {
```

Comment:
@maximbaz the replacements in iOS client are there for quite some years now, so I cannot give you a precise information about why one emoji was chosen over other ones. I think it is acceptable if in the web the replacement would be different from iOS, since the difference is subtle.